Parent/Parent/Dashboard/DashboardViewController.swift

```diff
@@ -73,7 +73,8 @@ class DashboardViewController: UIViewController, CustomNavbarProtocol {
                 currentStudentID = student.id
                 let color = ColorScheme.observee(student.id).color
                 alertsTabItem.badgeColor = color
-                navbarNameButton.setTitle(student.name, for: .normal)
+                let displayName = Core.User.displayName(student.name, pronouns: student.pronouns)
+                navbarNameButton.setTitle(displayName, for: .normal)
```

Comment:
Do we need to add these to the calendar and alerts screen as well?

Reply:
No, looks like this takes care of the other tabs.